### PR TITLE
chore: bump pg_tle version to 1.3.2

### PR DESCRIPTION
**Pull request by imor**
## What kind of change does this PR introduce?

- upgrade `pg_tle` from _v1.0.4_ to _v1.3.2_

## Additional context

Add any other context or screenshots.

## Action Items

- [x] **New extension releases** were Checked for any breaking changes
- [x] **Extensions compatibility** Checked
    * Proceed to [extensions compatibility testing](#extensions-compatibility-testing), mark as done after everything is completed
- [x] **Backup and Restore** Checked
    * Proceed to [backup testing](#backup-testing) while extensions are enabled
        - After every restore, re-run the tests specified at point [3.1](#extensions-compatibility-testing)

### Extensions compatibility testing

1. Enable every extension
    1. Check Postgres’ log output for any error messages while doing so
        1. This might unearth incompatibilities due to unsupported internal functions, missing libraries, or missing permissions
2. Disable every extension
    1. Check Postgres’ log output for any cleanup-related error messages
3. Re-enable each extension
    1. Run basic tests against the features they offer, e.g.:
        1. `pg_net` - execute HTTP requests
        2. `pg_graphql` - execute queries and mutations
        3. …to be filled in

### Backup Testing

Follow the testing steps steps for all the following cases:

- Pause on new Postgres version, restore on new Postgres version
- Pause on older Postgres version, restore on new Postgres version
- Run a single-file backup backup, restore the backup

#### Testing steps

1. Generate dummy data 
    * the ‘Countries’ or ‘Slack clone’ SQL editor snippets are decent datasets to work with, albeit limited
2. Save a db stats snapshot file
    * Do this by running `supa db-stats gather -p <project_ref>`
3. Backup the database, through pausing the project, or otherwise
4. Restore the backup, through unpausing the project or cli
5. Check the data has been recovered successfully
    1. Visual checks/navigating through the tables works
    2. Run `supa db-stats verify` against the project and the previously saved file

### CODE REVIEW

File: common.vars.pkr.hcl [hunk 1]
```diff
@@ -1 +1 @@
-postgres-version = "15.1.0.137"
+postgres-version = "15.1.0.138-pg_tle"
```
Comment (darora) [inline common.vars.pkr.hcl]: Please test on this branch and remove suffix once ready for merge

**Review by darora — COMMENTED** [review]
Please confirm that you've tested both pg_upgrades and pause/restores and ensured that the upgrade does not break either.

**Comment by imor** [comment]
> Please confirm that you've tested both pg_upgrades and pause/restores and ensured that the upgrade does not break either.

Test steps I performed:

1. Create a project on supabase.green with version [15.1.0.138-pg_tle](https://github.com/supabase/postgres/releases/tag/15.1.0.138-pg_tle).
2. Install `dbdev` in-db client.
3. Install [pg_idkit](https://database.dev/kiwicopple/pg_idkit) TLE version `0.0.4`.
4. Pause the project.
5. Restore the project. The restore succeeded and `pg_idkit` is reverted to the default version `0.0.1` as expected.
6. But for some reason the postgres role can't login after restore. Had to login as superuser and `alter role postgres login` to continue testing. Not sure what caused the postgres role's login to be disabled.
7. Try to upgrade to `15.1.0.137` (the only version shown for upgrade) but got this error: `{"message":"Project is not in a ready state to perform an upgrade"}`. Again not sure what caused it.

In summary, the original pg_tle pause/restore appears to have been fixed but I'm not sure what's causing these two issues:

1. `postgres` role can't login after restore.
2. Can't upgrade project to `15.1.0.137`.

@darora my project ref on supabase.green is `hbkcxauckeneyztwovoh` in case you can help figure out what caused these failures.

**Comment by darora** [comment]
Will do - I believe the restore has not completed.

**Review by olirice — APPROVED** [review]

**Comment by imor** [comment]
Hey @darora can you please approve this PR. It needs an approval from `supabase/backend` team.

**Comment by imor** [comment]
BTW the scope for this PR no longer includes the fixes for pause/restore. We want to upgrade `pg_tle` just for the support of allowing the `@` character in a TLEs name and fixing [another bug](https://github.com/supabase/dbdev/issues/159).

**Review by darora — APPROVED** [review]
